benchmark/opperf/nd_operations/nn_optimizer_operators.py

```diff
@@ -0,0 +1,68 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import mxnet as mx
+from benchmark.opperf.utils.benchmark_utils import run_op_benchmarks
+from benchmark.opperf.utils.op_registry_utils import get_all_optimizer_operators
+
+"""Performance benchmark tests for MXNet NDArray Optimizer Operators.
+
+1. Stochastic Gradient Descent (SGD)
+    1.1 multi_sgd_mom_update
+    1.2 mp_sgd_update
+    1.3 multi_sgd_update
+    1.4 sgd_mom_update
+    1.5 signsgd_update
+    1.6 mp_sgd_mom_update
+    1.7 multi_mp_sgd_update
+    1.8 multi_mp_sgd_mom_update
+    1.9 sgd_update
+3. signum_update
+4. rmspropalex_update
+5. ftml_update
+6. rmsprop_update
+7. ftrl_update
+8. adam_update
+"""
+
+
+def run_optimizer_operators_benchmarks(ctx=mx.cpu(), dtype='float32', warmup=25, runs=100):
```

Comment:
who calls this?

Reply:
my bad! added it to `opperf.py`
https://github.com/apache/incubator-mxnet/blob/0beea92e8319023f6e15a280b209befa2d919471/benchmark/opperf/opperf.py#L93